src/micromamba/config.cpp

```diff
@@ -184,6 +184,44 @@ set_config_describe_command(CLI::App* subcom)
     });
 }
 
+void
+set_config_get_command(CLI::App* subcom)
+{
+    auto& config = Configuration::instance();
+
+    auto& key = config.insert(Configurable("key", std::string())
+                                .group("cli")
+                                .rc_configurable(false)
+                                .description("Writes a new value for a key"));
+    subcom->add_option(
+        "key", key.set_cli_config({}), key.description());
+    YAML::Emitter out;
+
+    auto& specs = config.at("config_specs").get_wrapped<std::vector<std::string>>();
+    subcom->add_option("specs", specs.set_cli_config({}), specs.description());
+
+    subcom->callback([&]() {
+        auto& specs = config.at("config_specs").value<std::vector<std::string>>();
+
+    for (auto& group_it : config.get_grouped_config())
+    {
+        auto& positional_arg = key.compute().value();
+        auto& configs = group_it.second;
+
+        for (auto& c : configs)
+        {
+            if (c->name() == positional_arg)
+            {
+                std::cout << "🍉🦀\n";
+                detail::print_configurable(out, *c, false);
```

Comment:
I'm calling this function here because I need to get the value of a key `c` on this configurations dictionary. Is there a better way to do it?
This throws a segfault. Should I debug it deeper or do you have an idea of why it isn't working?

Reply:
I guess you get a dangling reference to the yaml emitter `out` that is define outside the callback.
You can also easily get the `ConfigurableInterface` by key in the `Configuration` using `at` as if it was a `std::map`:
```
subcom->callback([&]() {
    auto& c = config.at(key.compute().value());
    YAML::Emitter out;
    detail::print_configurable(out, c, false);
    std::cout << out.c_str() << std::endl;
});
```